test/controllers/api/v2/sync_plans_controller_test.rb

```diff
@@ -168,13 +168,13 @@ def test_add_products_protected
 
       assert_protected_action(:add_products, allowed_perms, denied_perms) do
         put :add_products, :id => @sync_plan.id, :organization_id => @organization.id,
-          :product_ids => @products.collect { |p| p.id }
+                           :product_ids => @products.collect { |p| p.id }
       end
     end
 
     def test_remove_products
       put :remove_products, :id => @sync_plan.id, :organization_id => @organization.id,
-          :product_ids => @products.collect { |p| p.id }
+                            :product_ids => @products.collect { |p| p.id }
 
```

Comment:
Yea, it's part of the hash that starts with `:id` so it should be lined up. That said, I don't have a strong preference for this rule though if we'd rather line up method arguments instead.